### DISCUSSION TITLE
Button

### DISCUSSION THREAD

**Discussion by Kentainfr** [requarks/wiki#6488] [Help / Questions] [answered]
Hey! Currently trying to learn how to use Wiki.JS and wanted to make a button like on the picture with Markdown. But seems like I can't or I don't know how. Any ideas on how to do it ? I feel like I'm missing something.

![image](https://github.com/requarks/wiki/assets/63867494/6a01fc94-00db-4b90-8361-d5726a27b6fe)

**Comment by P4mes4n** [comment, marked as answer]
[Im just gonna send you to this thread with the same question](https://github.com/requarks/wiki/discussions/6484#discussioncomment-6107609)
